queryFields.js

```diff
@@ -0,0 +1,139 @@
+// @ts-check
+
+/**
+ * Imports:
+ * @typedef {import('graphql').FieldNode} FieldNode
+ * @typedef {import('graphql').GraphQLResolveInfo} GraphQLResolveInfo
+ * @typedef {import('graphql').SelectionSetNode} SelectionSetNode
+ * @typedef {Object<string, any>} QueryObject
+ */
```

Comment:
What is this?

Reply:
Imports for type definitions for javascript. More info in the comment above.